templates/puppetdb-DLO-cleanup.service.epp

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Cleanup old discarded puppetdb reports
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/find /opt/puppetlabs/server/data/puppetdb/stockpile/discard/ -type f -mtime +90 -delete
```

Comment:
Also, one final note. For security reasons, this service should run as the [`PuppetDB user`](https://github.com/puppetlabs/puppetlabs-puppetdb#database_username)

Reply:
added user and group